tools/modules/AksEdgeDeploy/AksEdge-Arc.ps1

```diff
@@ -31,6 +31,12 @@ New-Variable -Option Constant -ErrorAction SilentlyContinue -Name azMinVersions
     "customlocation"   = "0.1.3"
     "k8s-extension"    = "1.3.3"
 }
+New-Variable -option Constant -ErrorAction SilentlyContinue -Name arcLocations -Value @(
+    "westeurope", "eastus", "westcentralus", "southcentralus", "southeastasia", "uksouth",
+    "eastus2", "westus2", "australiaeast", "northeurope", "francecentral", "centralus",
+    "westus", "northcentralus", "koreacentral", "japaneast", "eastasia", "westus3",
+    "canadacentral", "eastus2euap"
+)
 function Get-AideArcUserConfig {

```

Comment:
Is there a Test CLI we can use to test if creation of a resource would work? I'm asking because having this list maybe is not scalable in the future, and we will have to actively maintain it.